plugins/terminal.koplugin/whatspeed.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+echo  "Frequency in Hz: "
```

Comment:
Well, besides this brand new script. :rofl:
```suggestion
echo "Frequency in Hz: "
```

Reply:
This script is a leftover for testing the emulator. I will remove it.